CHANGELOG.md

```diff
@@ -100,6 +100,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 
 ## [Unreleased 2.x]
 ### Added
+- [SearchPipeline] Initial code for adding the SearchPhaseInjectorProcessor interface in Search Pipeline.([#7283](https://github.com/opensearch-project/OpenSearch/pull/7283))
```

Comment:
I am trying to find `SearchPhaseInjectorProcessor` but don't see it here, any hints how the `SearchPhaseInjectorProcessor` supposed to work so we could understand how this change fits into the large picture?

Reply:
Can you rewrite this in the imperative describing what it does? Like, "Add SearchPhaseResultsProcessor to search pipelines", or even "Add new processor interface to search pipelines to run between search phases"

The "initial code..." phrasing isn't clear about whether this adds any functionality or is just a building block for something that will come later.

Reply:
Updated the Changelog and also the title of this PR.